pages/linux/systemctl.md

```diff
@@ -1,6 +1,6 @@
 # systemctl
 
-> Control the systemd system and service manager.
+> Control the systemd system and service manager. You may need to run with `sudo` prefix to avoid permission denied issues.
```

Comment:
I don't think this is a good recommendation. Using `sudo systemctl` has [known problems](https://stackoverflow.com/q/34167257/5637701).

At a minimum, we should include some qualifications and/or warnings.

Reply:
thank you for your sharing that issue, I think this old explanation is enough now.